### ISSUE TITLE
ember cli shims

### ISSUE THREAD

**Issue by billybonks** [mattmcmanus/atom-ember-snippets#15]
Hi would you want to migrate your snippets to be ember-cli-shims format ?

so instead of 

```
import Ember from 'ember'

export default Ember.component.extend({

})
```

it would be

```
import Component from 'ember-component'

export default Component.extend({

})
```

**Comment by seifsallam** [comment]
@billybonks I'm not up-todate with Ember syntax, as I no longer program, but it sounds good! Feel free to create a pull request and i'll merge it.

**Comment by Exelord** [comment]
Deprecated in favor of https://github.com/ember-cli/ember-rfc176-data

**Comment by mattmcmanus** [comment]
I just shipped `2.1.0`, which includes the work done in #26

**Comment by billybonks** [comment]
nice :)
